GOVERNANCE.md

```diff
@@ -142,3 +142,13 @@ Maintainer in question). This can be due to lack of participation or conduct
 violations, among other things. Note that Maintainers are subject to a higher
 set of behavioral and communicative standards than average contributor or
 participant.
+
+### Single-Company Influence
+
+At most 1/3 of the maintainers may be paid for their contributions by a single
+company. This is to reduce the risk of a single company controlling the
+project's direction. If the 1/3 limit gets exceeded because an existing
+maintainer gets hired by the same company as some other existing maintainer(s),
+then the maintainers will have to decide how to resolve the situation. The
+maintainer in question gets to vote (the company will still not have half of
+the votes).
```

Comment:
Strictly speaking, the parenthetical is only true if there are at least 5 maintainers.

Reply:
Makes sense. Done.